node/src/components/network/outgoing.rs

```diff
@@ -27,8 +27,8 @@
 //! * The `Dialer` is expected to produce `DialOutcomes` for every dial [`DialRequest::Dial`]
 //!   eventually. These must be forwarded to the `OutgoingManager` via the `handle_dial_outcome`
 //!   function.
-//! * The `perform_housekeeping` method must be called periodically to give the
-//!   `OutgoingManager` a chance to initiate reconnections and collect garbage.
+//! * The `perform_housekeeping` method must be called periodically to give the `OutgoingManager` a
+//!   chance to initiate reconnections and collect garbage.
```

Comment:
Can you revert this change as unrelated? Or at least put it in a separate commit (assuming the formatter keeps insisting that this is the correct layout).

Reply:
yeah this was the formatter, I'll move it to another commit 👍